### ISSUE TITLE
How change the map style?

### ISSUE THREAD

**Issue by fabiana001** [gmplot/gmplot#81]
[Google Maps API](https://developers.google.com/maps/documentation/javascript/styling) allow us to change the style of the map adding a style json. Can I personalize the map style, adding this json as input parameter to the gmplot.GoogleMapPlotter function? 

Example:
From this:
![screenshot from 2018-08-08 18-24-31](https://user-images.githubusercontent.com/2932983/43850605-af9ade32-9b38-11e8-860c-f8ffccbd760b.png)

To this:
![screenshot from 2018-08-08 18-06-13](https://user-images.githubusercontent.com/2932983/43850225-a6197950-9b37-11e8-8402-e91ab568a281.png)

**Comment by averysmith10** [comment]
interested in this as well

**Comment by frslm** [comment]
@fabiana001 and @AveryData, this was just merged in #148. You can now customize your maps using the `map_styles` option:
```python
map_styles = [
    {
        'featureType': 'all',
        'stylers': [
            {'saturation': -80},
            {'lightness': 60},
        ]
    }
]

apikey='' # (you'll need a valid API key to customize map styles)
gmap = gmplot.GoogleMapPlotter(37.76, -122.43, 13, apikey=apikey, map_styles=map_styles)
```
